app/controllers/openid_connect/logout_controller.rb

```diff
@@ -48,6 +55,13 @@ def delete
       if result.success? && redirect_uri
         handle_logout(result, redirect_uri)
       else
+        if result.extra[:integration_errors].present?
```

Comment:
issue: The method `AnalyticsEvents#oidc_logout_submitted` tracks the 'OIDC Logout Submitted' event. If there was any need to correlate events in CloudWatch, this won't work as is.